tetrateci/setup_boring_go.sh

```diff
@@ -1,13 +1,28 @@
-#!/bin/bash
+#!env bash
+set -e
+
+if $(grep "1.7" <<< $TAG ); then
```

Comment:
Can you check if these 3 go versions are available in boring_go?

Reply:
Yes they do, :)